Hmm, when merging the main branch that includes the JVM build (#13), CI is now failing:
https://github.com/CUTR-at-USF/opentripplanner-client-library/runs/2298307398?check_suite_focus=true

with:

```
...
[Step 1 of 1] Preparing commonized Kotlin/Native libraries for targets [ios_x64], [ios_arm64] (280 items): Done


e: Could not find "kotlinx-coroutines-core_concurrentMain" in [/Users/runner/work/opentripplanner-client-library/opentripplanner-client-library, /Users/runner/.konan/klib, /Users/runner/.konan/kotlin-native-prebuilt-macos-1.4.32/klib/common, /Users/runner/.konan/kotlin-native-prebuilt-macos-1.4.32/klib/platform/ios_arm64].
Daemon vm is shutting down... The daemon has exited normally or was terminated in response to a user interrupt.
----- End of the daemon log -----


FAILURE: Build failed with an exception.

* What went wrong:
Gradle build daemon disappeared unexpectedly (it may have been killed or may have crashed)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

> Task :library:compileIosMainKotlinMetadata
e: Could not find "kotlinx-coroutines-core_concurrentMain" in [/Users/runner/work/opentripplanner-client-library/opentripplanner-client-library, /Users/runner/.konan/klib, /Users/runner/.konan/kotlin-native-prebuilt-macos-1.4.32/klib/common, /Users/runner/.konan/kotlin-native-prebuilt-macos-1.4.32/klib/platform/ios_arm64].
Error: Process completed with exit code 1.
```

--- a/library/build.gradle.kts
+++ b/library/build.gradle.kts
@@ -61,7 +61,7 @@ kotlin {
         val jvmMain by getting {
             dependencies {
                 implementation("io.ktor:ktor-client-java:$ktorVersion")
-                implementation ("org.jetbrains.kotlinx:kotlinx-coroutines-swing:1.4.2")
+                implementation ("org.jetbrains.kotlinx:kotlinx-coroutines-swing:$coroutineVersion")
             }
         }
     }

--- a/library/build.gradle.kts
+++ b/library/build.gradle.kts
@@ -14,10 +14,6 @@ plugins {
 group = "edu.usf.cutr.otp"
 version = "1.0.0"
 
-repositories {
-    mavenLocal()
-}
-
 kotlin {
     android()
     jvm {

--- a/library/build.gradle.kts
+++ b/library/build.gradle.kts
@@ -1,6 +1,6 @@
 
-val ktorVersion = "1.5.0"
-val coroutineVersion = "1.4.2"
+val ktorVersion = "1.5.3"
+val coroutineVersion = "1.4.3"
 
 plugins {
     kotlin("multiplatform")

